test/perf/dotnet-tools/BenchmarkDotNet.Extensions/DiffableDisassemblyExporter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Disassemblers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    // a simplified copy of internal BDN type: https://github.com/dotnet/BenchmarkDotNet/blob/0445917bf93059f17cb09e7d48cdb5e27a096c37/src/BenchmarkDotNet/Disassemblers/Exporters/GithubMarkdownDisassemblyExporter.cs#L35-L80
+    internal static class DiffableDisassemblyExporter
```

Comment:
this type is a set of hacks that we have temporarily added to the performance repo and never had the time to provide a proper solution. IMO you should not be copying it and we should just make these types public in BenchmarkDotNet if [DisassemblyDiagnoser](https://adamsitnik.com/Disassembly-Diagnoser/) does not meet your needs.

Reply:
Oh, didn't know that. Yeah, I don't think we would need anything beyond what `DisassemblyDiagnoser` supports.